src/shared/Notifications/Notifications.scss

```diff
@@ -16,6 +23,14 @@
   border-radius: 5px;
   cursor: pointer;
   line-height: 1.2em;
+  
+  &__ntf-box {
+    @include flex($justify: space-around, $wrap: nowrap, $align: center);
+  }
+
+  &__title {
+    text-transform: uppercase;
+  }
 
   &.change {
     background-color: #444;
```

Comment:
Could you please add this color to our scss styles default file and then use it as a variable here?

Reply:
done)